flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/ExpressionEditorService.java

```diff
@@ -251,6 +253,32 @@ public CompletableFuture<FunctionCallTemplateResponse> functionCallTemplate(Func
         });
     }
 
+    @JsonRequest
+    public CompletableFuture<SuccessResponse> importModule(ImportModuleRequest request) {
+        return CompletableFuture.supplyAsync(() -> {
+            SuccessResponse response = new SuccessResponse();
+            try {
+                String fileUri = CommonUtils.getExprUri(request.filePath());
+                Optional<Document> document = workspaceManagerProxy.get(fileUri).document(Path.of(request.filePath()));
+                if (document.isPresent()) {
+                    ExpressionEditorContext expressionEditorContext = new ExpressionEditorContext(
+                            workspaceManagerProxy.get(fileUri),
+                            Path.of(request.filePath()), document.get());
+                    String importStatement = request.importStatement()
+                            .replaceFirst("^import\\s+", "")
+                            .replaceAll(";\\n$", "");
+                    Optional<TextEdit> importTextEdit = expressionEditorContext
+                            .getImport(importStatement);
+                    importTextEdit.ifPresent(textEdit -> expressionEditorContext.applyTextEdits(List.of(textEdit)));
+                    response.setSuccess();
+                }
+            } catch (Exception e) {
+                response.setError(e);
```

Comment:
Shouldn't we set `success` property of the response as `false` in this case?

Reply:
Didn't set explicitly since the default value is false

Reply:
It seems default value is not defined. maybe I'm missing something?

<img width="501" alt="Screenshot 2025-01-05 at 20 59 47" src="https://github.com/user-attachments/assets/730a4be2-8ffc-40f7-b13e-c9815bf0933d" />